hooks/006-add-gdm.chroot

```diff
@@ -82,6 +82,7 @@ rm -f /etc/systemd/user/default.target.wants/pipewire*
 rm -f /usr/lib/systemd/user/org.freedesktop.IBus.session.GNOME.service
 rm -f /usr/lib/systemd/user/org.freedesktop.IBus.session.generic.service
 rm -f /usr/lib/systemd/user/gnome-session.target.wants/org.freedesktop.IBus.session.GNOME.service
+rm -f /usr/lib/systemd/user/colord-session.service
```

Comment:
What about `/lib/systemd/system/colord.service`?

Reply:
That's not provided by ubuntu-desktop-session.  I think the current issue is just the user session service which we want to dbus activate in ubuntu-desktop-session